### PR TITLE
Lighthouse UI changes and fixed broken test of LH

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes/Fixes/Resolves

1370

# Explain the feature/fix

1. Implement ui changes in lighthouse SPA
2. Fixed broken unit tests of angular

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

Screenshot 2021-08-02 at 2 09 53 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Review by ghanlohar — APPROVED** [review]
LGTM

File: packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            *ngIf="!loading && isEmpty">
+            <div class="pf-c-empty-state pf-m-xs">
+                <div class="pf-c-empty-state__content">
+                    <i class="fas fa-cubes pf-c-empty-state__icon" aria-hidden="true"></i>
+                    <h1 class="pf-c-title pf-m-md">
+                        No data found
+                    </h1>
+                    <div class="pf-c-empty-state__body">
+                        Contact<a href="mailto:one-portal-devel@redhat.com">
+                            one-portal-devel@redhat.com
```
Comment (riginoommen) [inline packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.html:48]: Make this as an env variable

File: packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        this.isEmpty = data.listLHProperties.length === 0;
+      });
+  }
+
+  validateUrl(url: string) {
+    this.validUrl = url.indexOf('http://') == 0 || url.indexOf('https://') == 0;
+  }
+
+  navigateToReportGeneration() {
+    // this.router.navigateByUrl(`/?site_url=${this.sites}`);
```
Comment (riginoommen) [inline packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.ts:52]:
```suggestion
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.ts (hunk 2 above)
Comment (akhilmhdh) [reply packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.ts]: Done

File: packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.html (hunk 1 above)
Comment (akhilmhdh) [reply packages/lighthouse-spa/src/app/dashboard/pages/home/home.component.html]: Done

**Review by riginoommen — APPROVED** [review]

**Pull request merged**